time-to-live.md

```diff
@@ -215,9 +210,18 @@ TiDB 会定时采集 TTL 的运行时信息，并在 Grafana 中提供了相关
 
   其中列 `table_id` 为分区表 ID，而 `parent_table_id` 为表的 ID，与 `infomation_schema.tables` 表中的 ID 对应。`table_schema`、`table_name`、`partition_name` 分别对应表示数据库、表名、分区名。`create_time`、`finish_time`、`ttl_expire` 分别表示 TTL 任务的创建时间、结束时间和过期时间。`expired_rows` 与 `deleted_rows` 表示过期行数与成功删除的行数。
 
-## 工具兼容性
+## 工具和功能兼容性
+
+TTL 功能能够与 TiDB 的迁移、备份、恢复工具和功能一同使用。
 
-作为实验特性，TTL 特性暂时不兼容包括 BR、TiDB Lightning、TiCDC 在内的数据导入导出以及同步工具。
+| 工具/功能名称 | 最低兼容版本 | 说明 |
+| --- | --- | --- |
+| Backup & Restore (BR) | 6.6 | 恢复时会自动将表的 `TTL_ENABLE` 属性设置为 `OFF`，关闭 TTL。 |
+| TiDB Lightning | 6.6 | 导入后如果表中有 TTL 属性，会自动将表的 `TTL_ENABLE` 属性设置为 `OFF`，关闭 TTL。 |
+| Dumpling | 6.6 | |
+| TiCDC | 6.6 | 下游表的 `TTL_ENABLE` 属性将被强制设置为 `OFF`。 |
+| TiDB Binlog | 7.0 | Drainer 向下游为 TiDB/MySQL 数据库时表的 `TTL_ENABLE` 属性将被强制设置为 `OFF`。 |
+| Flashback | | Flashback 一个数据库或表时，它们的 `TTL_ENABLE` 将被强制设置为 `OFF`。Flashback 整个集群时，系统变量 `TTL_JOB_ENABLE` 将被设置为 `OFF`，同时表的 `TTL_ENABLE` 属性将保持原样。 |
```

Comment:
@lcwangchao PTAL.

Binlog compatibility is still WIP. It's a minor change: https://github.com/pingcap/tidb-binlog/pull/1219

Reply:
I think we can remove dumpling and binlog. Dumpling is naturally compatible with TTL. Binlog is not longer supported.